packages/shared/index.ts

```diff
@@ -42,6 +42,17 @@ export function buildBlockExplorerTxURL({
   return `${networkUrl.url}tx/${txHash}`;
 }
 
+export function buildExplorerTokenURL({
```

Comment:
to keep the convention
```suggestion
export function buildBlockExplorerTokenURL({
```